From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Tue, 3 Mar 2020 22:53:35 -0800
Subject: ANDROID: gki_defconfig: Enable MFD_SYSCON on x86

This option was accidentally skipped when it was added on arm64.

Bug: 144867487
Change-Id: Ifa87a894954ec9a26d5ab40e7d18e2f2f5e4f416
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/x86/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -265,6 +265,7 @@ CONFIG_GPIOLIB=y
 CONFIG_DEVFREQ_THERMAL=y
 # CONFIG_X86_PKG_TEMP_THERMAL is not set
 CONFIG_WATCHDOG=y
+CONFIG_MFD_SYSCON=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
